src/Orc.Controls/Controls/DateTimePicker/DateTimePicker.cs

```diff
@@ -1675,6 +1778,30 @@ private CultureInfo GetCulture()
         {
             return Culture ?? CultureInfo.CurrentUICulture;
         }
+
+        private void TryProceedSecondMinuteEditing(TextBox secondOrMinuteTextBox)
+        {
+            if (_suspendTextChanged)
+            {
+                return;
+            }
+
+            var text = secondOrMinuteTextBox.Text;
+            if (string.IsNullOrWhiteSpace(text))
+            {
+                return;
+            }
+
+            if (new HashSet<string> { "1", "2", "3", "4", "5" }.Contains(text))
```

Comment:
Please create static hashset, we don't have to re-initialize a new array / hashset every time.

Reply:
fixed